### ISSUE TITLE
Can't open particle options in mod menu

### ISSUE THREAD

**Issue by 1GamingFox** [PigCart/particle-rain#86]
I've tried clicking them in every way I can think of but they won't open

**Comment by PigCart** [comment]
game log? seeing a video of this would help also

**Comment by 1GamingFox** [comment]
Yes, I clicked on them, it just doesn't register for whatever reason. Thank you for your time looking into it.
